Confidence/src/main/java/com/spotify/confidence/Confidence.kt

```diff
@@ -144,7 +146,7 @@ class Confidence internal constructor(
             it.getContext().filterKeys { key -> !removedKeys.contains(key) } + contextMap.value
         } ?: contextMap.value
 
-    override fun withContext(context: Map<String, ConfidenceValue>): Confidence = Confidence(
+    override fun withContext(context: Map<String, ConfidenceValue>): ConfidenceAPI = Confidence(
```

Comment:
The `EventSender` interface already extends `Contextual`: can we just return the `EventSender` type from withContext?

Reply:
I think we can good point.